docs/openapi.yaml

```diff
@@ -599,12 +603,13 @@ paths:
       - name: hash
         in: path
         description: Hash of the block
+        example: hash=0xbf06705d0b8c2389eb4b3a19ce72096b7158b158e2cd4ceb27d8de9db19bb3be
```

Comment:
IIUC, `example` should contain literals which are syntactically valid https://swagger.io/specification/#example-object, so in this case it would just be `0xbf06705d0b8c2389eb4b3a19ce72096b7158b158e2cd4ceb27d8de9db19bb3be`.

E.g. some tooling like OpenAPI front-ends that have "try me" functionality will often use the `example` as a default value. 

There's a few other places where it looks like this change is also needed.